ansible_collections/arista/avd/roles/eos_designs/doc/fabric-variables.md

```diff
@@ -14,13 +14,22 @@
   - IBGP (only with OSPF or ISIS variants in underlay)
 - Only summary network addresses need to be defined. IP addresses are then assigned to each node, based on its unique device id.
   - To view IP address allocation and consumption, a summary is provided in the auto-generated fabric documentation in Markdown and CSV format.
-- The variables should be applied to all devices in the fabric.
 
 *Only supported with core_interfaces data model.
 
+## Flagging a Device as Not Deployed
+- It is possible to provision configurations for a complete topology but flag devices as undeployed using the host level variable `is_deployed: false`. I.e. via the Ansible inventory file.
+- By default, this will have no impact, except cause the device to be ignored by CloudVision when used in conjunction with the `eos_config_deploy_cvp` role. If the device is not present in the network, `eos_validate_state` role will fail tests on peers of the undeployed device trying to verify that interfaces are up.
```

Comment:
so here it means that `is_deployed` would not have any effect if `eos_config_deploy_cli` is used and the config would still be deployed - is it correct? If so probably make it clear the behavior when using the `eos_config_deploy_cli` role.

Reply:
Agreed and I've attempted to make the documentation clearer